### ISSUE TITLE
Current year in date-picker got AVT violation that it's not in landmark element

### ISSUE THREAD

**Issue by lj7720** [IBM/carbon-components-angular#1743]
## Detailed description

> Describe in detail the issue you're having.
Accessibility checker got violation against current year Data Picker that "Content is not within a landmark element".

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug

> Is this issue related to a specific component?
date picker

> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in?
Firefox 85.0.2

> What version of the Carbon Design System are you using?
3.54.2

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
ADDI

## Steps to reproduce the issue

1. Open page containing date picker element
2. Click to open dialog of date picker, run Accessibility Checker tool.

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/74904663/109267829-2abc6b80-7845-11eb-9eb5-e295688cde3c.png)

* Notes
